code/modules/vehicles/quad.dm

```diff
@@ -28,8 +30,10 @@
 /obj/vehicle/train/engine/quadbike/Initialize()
 	cell = new /obj/item/cell/high(src)
 	key = new key_type(src)
+	soundloop = new(list(src), FALSE)
 	. = ..()
```

Comment:
As a train, this parent call _appears_ to invoke the following:
```
/obj/vehicle/train/Initialize()
	. = ..()
	for(var/obj/vehicle/train/T in orange(1, src))
		latch(T, null)
```
Can you test what happens if a quadbike is spawned in adjacent to another? (Or to other train-type things?)

Reply:
It does indeed attach stuff that's spawned in such that they meet the latch proc requirements (/train type, facing the same way) which seems intentional with quads/trailer, but have added a check so it can not do that for train subtypes, and won't for snowmobiles.